routes/index.js

```diff
@@ -370,13 +370,16 @@ Routes.prototype.proxyRouteOpenhab = function (req, res) {
     delete requestHeaders['x-forwarded-for'];
     delete requestHeaders['x-forwarded-proto'];
     delete requestHeaders['connection'];
-    requestHeaders['host'] = req.headers.host || system.getHost() + ':' + system.getPort();
+    requestHeaders['host'] = req.headers["x-forwarded-host"] || system.getHost() + ':' + system.getPort();
     requestHeaders['user-agent'] = 'openhab-cloud/0.0.1';
     // Strip off path prefix for remote vhosts hack
     var requestPath = req.path;
     if (requestPath.indexOf('/remote/') === 0) {
         requestPath = requestPath.replace('/remote', '');
         // TODO: this is too dirty :-(
+        if(requestPath.indexOf('/remote/auth') !== 0){
+            delete requestHeaders['authorization'];
```

Comment:
Just an FYI, the openHAB 3 ui will automatically use another header for authentication `X-OPENHAB-AUTH-HEADER`

Reply:
apparently this is no needed. After i reboot the server that is hosting openhab-cloud instance. Everything is working as intended. Thanks for your time.